packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -496,6 +502,14 @@ export async function activate(context: vscode.ExtensionContext) {
     return internalApi;
   }
 
+  // Set functions enabled context
+  const functionsEnabled = sfdxCoreSettings.getFunctionsEnabled();
```

Comment:
If user updates the setting, it won't be applied unless restarted. We could use`workspace.onDidChangeConfiguration` to listen and updates the context

Reply:
I think the purpose of this config is to deter users from turning it on. Once we have reached the beta stage, we will move this to package.json like the other experimental configs. So we are good with it as is I think.